azurerm/resource_arm_image_test.go

```diff
@@ -1388,3 +1433,14 @@ resource "azurerm_virtual_machine_scale_set" "testdestination" {
 }
 `, rInt, location, rInt, rInt, rInt, hostName, rInt, rInt, userName, password, rInt, userName, password, rInt)
 }
+
+// Region list pulled from https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy-zrs
+func supportsZRS(location string) bool {
+	zrsRegions := []string{"westus2", "francecentral", "southeastasia", "westeurope", "northeurope", "japaneast", "uksouth", "eastus", "eastus2", "centralus"}
```

Comment:
this only covers Azure Public (there's different Azure Regions which are supported e.g. Government, China, Germany) - as such I think we'd be best to remove this for the moment? (we're running the tests in some of these regions)

Reply:
Sure, my concern had been that I did not know in which locations these tests were going to be run, and did not want to have an acceptance test failure simply due to the fact that ZRS storage is not supported in some locations.